main.tf

```diff
@@ -42,6 +42,16 @@ resource "azurerm_cosmosdb_account" "db" {
     max_interval_in_seconds = try(var.cosmosdb.consistency_policy.max_interval_in_seconds, 300)
     max_staleness_prefix    = try(var.cosmosdb.consistency_policy.max_staleness_prefix, 100000)
   }
+
+  ip_range_filter = var.cosmosdb.ip_range_filter
```

Comment:
ip_range_filter should be optional too, using the try function. The default value should be null

Reply:
I changed it :) it it okay like this?